include/dlr_data_transform.h

```diff
@@ -80,16 +82,46 @@ class DLR_DLL DateTimeTransformer : public Transformer {
                    DLDataType dtype, DLContext ctx, tvm::runtime::NDArray& input_array) const;
 };
 
+class DLR_DLL TextTransformer : public Transformer {
+ public:
+  TextTransformer();
+
+  virtual void MapToNDArray(const nlohmann::json& input_json, const nlohmann::json& transform,
+                            tvm::runtime::NDArray& input_array) const override;
+
+  virtual void InitNDArray(const nlohmann::json& input_json, const nlohmann::json& transform,
+                           DLDataType dtype, DLContext ctx,
+                           tvm::runtime::NDArray& input_array) const override;
+
+  inline void SetIndex(int idx) const { column_idx = idx; };
+
+ private:
+  const static int kCharNum = 256;
+  char delims[kCharNum];
+  std::unique_ptr<std::vector<std::unordered_map<std::string, int>>> vocab_to_cols;
```

Comment:
Let's rename class member variables to end with `_` to follow our conventions, for example: `vocab_to_cols_`

Reply:
Done.